### PR TITLE
Add "Copy to clipboard" button. 

### PR DESCRIPTION
**Pull request by Secrus**
This code allows for adding the "copy to clipboard" button using Hugo shortcodes. As not every code block needs to be copyable, the button will be added only to parts wrapped in `{{% clipboard %}} ... {{% /clipboard %}}` tags. After this is accepted and merged, I will prepare separate PR in the main repo, adding required tags to docs. 

### How to preview
As docs files are not kept with website code, previewing requires some legwork. Checkout this PR, setup the website (as in README) and manually add required tags in downloaded docs files.

### CODE REVIEW

File: src/js/controllers/clipboard_controller.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import { Controller } from "stimulus"
+
+export default class extends Controller {
+  svgCopy =
```
Comment (neersighted) [inline src/js/controllers/clipboard_controller.js:4]: Not sure I like the SVG being inlined in the code (the build process could always bundle it) -- also, what is the licensing on these icons?

**Review by neersighted — CHANGES_REQUESTED** [review]

File: src/js/controllers/clipboard_controller.js (hunk 1 above)
Comment (Secrus) [reply src/js/controllers/clipboard_controller.js:4]: Will look into the SVG inline. 

When it comes to licensing, I don't really know. I used [this article](https://simplernerd.com/hugo-add-copy-to-clipboard-button/) as a reference and the author uses those (both in the article itself and on his page).

Comment (neersighted) [reply src/js/controllers/clipboard_controller.js:4]: Okay, let's avoid SVG images of unknown provenance. Could you please find some permissively licensed icons and either add them as a dep, or add them as files with license comments and inline them in the build?

Comment (Secrus) [reply src/js/controllers/clipboard_controller.js]: Changed SVGs to ones from Google Fonts (free to use in open source, checked) and moved them to files imported to Controller.

Comment (neersighted) [reply src/js/controllers/clipboard_controller.js]: Code looks good -- let's just add comments to the SVG files with the license.

**Review by neersighted — CHANGES_REQUESTED** [review]
Code looks good -- let's just add comments to the SVG files with the license.

**Comment by branchvincent** [comment]
fyi, `master` is now using node 16. also, you can try out running locally with just `make site`

**Comment by Secrus** [comment]
@neersighted added licensing comments to SVGs

**Comment by b-kamphorst** [comment]
Hi @abn! I intended to work on https://github.com/python-poetry/poetry/issues/5619 using this MR, but I now see that the changes have been reverted. Could you elaborate on the revert? In particular: are there certain changes that need to be made for this MR to be acceptable?

**Comment by Secrus** [comment]
> Hi @abn! I intended to work on [python-poetry/poetry#5619](https://github.com/python-poetry/poetry/issues/5619) using this MR, but I now see that the changes have been reverted. Could you elaborate on the revert? In particular: are there certain changes that need to be made for this MR to be acceptable?

Hi! I am not Abn, but I will try to answer. Biggest issue was that with current setup there was no good plugin for rollup to bundle SVG files with icons for the button. It looked ok on previews, but when we deployed it to production, it broke all JS on our website and that caused a lot of issues. Feel free to work on that feature.

**Comment by b-kamphorst** [comment]
> > Hi @abn! I intended to work on [python-poetry/poetry#5619](https://github.com/python-poetry/poetry/issues/5619) using this MR, but I now see that the changes have been reverted. Could you elaborate on the revert? In particular: are there certain changes that need to be made for this MR to be acceptable?
> 
> Hi! I am not Abn, but I will try to answer. Biggest issue was that with current setup there was no good plugin for rollup to bundle SVG files with icons for the button. It looked ok on previews, but when we deployed it to production, it broke all JS on our website and that caused a lot of issues. Feel free to work on that feature.

OK. I have to admit that I am completely novel to hugo and JS, but this might be a nice time to get started (low prio though). Do you have any thoughts on (1) how to proceed and, perhaps more importantly, (2) how to test (and actually fail) before production?

**Pull request merged**